pkg/actuators/machine/reconciler.go

```diff
@@ -251,6 +258,25 @@ func (r *Reconciler) update() error {
 	return r.requeueIfInstancePending(newestInstance)
 }
 
+func (r *Reconciler) getTagsFromInfrastructure() (map[string]string, error) {
+	infra := &configv1.Infrastructure{}
+	infraName := client.ObjectKey{Name: awsclient.GlobalInfrastuctureName}
+
+	if err := r.client.Get(r.Context, infraName, infra); err != nil {
+		return nil, err
+	}
+	tags := make(map[string]string)
+	ok, resourceTags := fetchInfraResourceTags(infra)
```

Comment:
Convention in go is to return the resource and then the `ok`, so I would be tempted to switch there around

Reply:
Sure , point taken